### ISSUE TITLE
Not compatible with NP 8.6.7

### ISSUE THREAD

**Issue by akniesel** [michaelxzhang/Npp-Highlighter#31]
Not compatible with NP 8.6.7

Before it was working

**Comment by suchyindustries** [comment]
Yeah I miss this plugin. Why is it incompatible

**Comment by akniesel** [comment]
NPP does not allow to install it, got an error message that it is incompatible. Is there any chance to update it?

**Comment by SimonSteinberger** [comment]
Yes, please update the plugin for the latest NPP versions :-))) This plugin is so extremely useful!

**Comment by N1ck01** [comment]
If this happens to help anyone, I got this plugin working with Notepad++ v8.8.1 (64-bit) by creating a folder named "NppHighlighter" WITHOUT the dash character AND renaming the .dll file to "NppHighlighter.dll" WITHOUT the dash character into the "plugins" folder:
`C:\Program Files\Notepad++\plugins\NppHighlighter\NppHighlighter.dll`

**Comment by SimonSteinberger** [comment]
> If this happens to help anyone, I got this plugin working with Notepad++ v8.8.1 (64-bit) by creating a folder named "NppHighlighter" WITHOUT the dash character AND renaming the .dll file to "NppHighlighter.dll" WITHOUT the dash character into the "plugins" folder: `C:\Program Files\Notepad++\plugins\NppHighlighter\NppHighlighter.dll`

Doesn't work on my 8.8.1 *32-bit* version, sadly. Npp crashes on startup.

**Comment by TobiasKnauss** [comment]
> If this happens to help anyone, I got this plugin working with Notepad++ v8.8.1 (64-bit) by creating a folder named "NppHighlighter" WITHOUT the dash character AND renaming the .dll file to "NppHighlighter.dll" WITHOUT the dash character into the "plugins" folder: `C:\Program Files\Notepad++\plugins\NppHighlighter\NppHighlighter.dll`

Still valid for 8.9.6.2. (64bit)
